rotkehlchen/chain/manager.py

```diff
@@ -1680,12 +1685,22 @@ def get_eth2_history_events(
             if stats_entry.pnl_balance.amount == ZERO:
                 continue
 
-            if stats_entry.pnl_balance.amount > ZERO:
+            # Take into account the validator ownership proportion if is not 100%
+            validator_ownership = validators_ownership.get(stats_entry.validator_index, FVal(1))
+            if validator_ownership != FVal(1):
+                pnl_balance = Balance(
```

Comment:
nitpick: Not sure if we need it anywhere else, but this would be a good addition to the `Balance()` class. A multiplication by an int/Fval. Not really needed though  ... just a thought in case we do this in other places too.